packages/neon/neon/lib/src/pages/login.dart

```diff
@@ -31,7 +31,9 @@ class _LoginPageState extends State<LoginPage> {
 
   void login(final String url) {
     if (_formKey.currentState!.validate()) {
-      LoginCheckServerStatusRoute(serverUrl: url).go(context);
+      final uri = Uri.parse(url.endsWith('/') ? url.substring(0, url.length - 1) : url);
```

Comment:
I think creating an extension for Uri that removes the trailing slash would make this a lot better.

Reply:
I'll leave this open for now and we'll see once the mentioned FilesUri becomes a thing